docs/content/about/contribute.md

```diff
@@ -70,4 +78,4 @@ We are using GitHub Issues for our public bugs. We keep a close eye on this and
 
 ### Reporting New Issues
 
-The best way to get your bug fixed is to provide a reduced test case. You can get inspiration from our current [test suite](https://github.com/adaltas/node-nikita/tree/master/test). Some test require a specific environment which is provided through [docker environments](https://github.com/adaltas/node-nikita/tree/master/docker).
+The best way to get your bug fixed is to provide a reduced test case. You can get inspiration from our current test suites, e.g., [the tests](https://github.com/adaltas/node-nikita/tree/master/packages/engine/test) of the "engine" package. Some test require a specific environment which is provided through [Docker or LXD environments](https://github.com/adaltas/node-nikita/tree/master/packages/engine/env).
```

Comment:
replace reduce by minimal. Start the last sentence with `Note, tests are filtered by tag and some tests require..`